pkg/metrics/metrics.go

```diff
@@ -47,7 +48,7 @@ func NewServer(log *logrus.Entry) *Metrics {
 	containerImageVersion := promauto.With(reg).NewGaugeVec(
 		prometheus.GaugeOpts{
 			Namespace: "version_checker",
-			Name:      "is_latest_version",
+			Name:      "version_checker_is_latest_version",
```

Comment:
The `version_checker` prefix comes from the `Namespace` field,

```suggestion
			Name:      "is_latest_version",
```